qml/Keyboard.qml

```diff
@@ -46,7 +48,7 @@ Item {
     property bool cursorSwipe: false
     property int prevSwipePositionX
     property int prevSwipePositionY
-    property int cursorSwipeDuration: 400
+    property int cursorSwipeDuration: 5000
```

Comment:
Is there any reason to have this time out now that there is a "Done" button?

Reply:
I decided to still have a timeout albeit quite longer than before. Honestly, I'm not sure if it still make sense or not but I think it's fine leaving it there.